corehq/apps/locations/views.py

```diff
@@ -164,7 +164,10 @@ def mk_loctype(name, parent_type, administrative,
             if loc_type is None:
                 loc_type = LocationType(domain=self.domain)
             loc_type.name = name
-            loc_type.administrative = administrative
+            # only CommTrack domains should have non-administrative locations
+            loc_type.administrative = (administrative
```

Comment:
It comes from the client, so this could be set in javacript.  Maybe if I put this right where the parameters are pulled from the request object?